scripts/superbuild/examples/cuda-distconv.sh

```diff
@@ -107,12 +115,13 @@ cmake \
     -D LBANN_SB_BUILD_Aluminum=${BUILD_LBANN_STACK} \
     -D LBANN_SB_Aluminum_CXX_FLAGS="${EXTRA_CXX_FLAGS}" \
     -D LBANN_SB_Aluminum_CUDA_FLAGS="${EXTRA_CUDA_FLAGS}" \
-    -D LBANN_SB_FWD_Aluminum_ALUMINUM_ENABLE_CALIPER=ON \
+    -D LBANN_SB_FWD_Aluminum_ALUMINUM_ENABLE_CALIPER=OFF \
```

Comment:
May as well set `LBANN_SB_BUILD_adiak=OFF` and `LBANN_SB_BUILD_Caliper=OFF`, too.